src/types/onyx/Transaction.ts

```diff
@@ -0,0 +1,22 @@
+import * as OnyxCommon from './OnyxCommon';
+
+type Comment = {
+    comment?: string;
+};
```

Comment:
NAB, do we really need a type for Comment?

Reply:
From TS guidelines (TS_STYLE.md): 

> If T isn't a primitive type, create a separate type for the T object structure of your prop and use it.

Reply:
I'm not sure this is the case here. Comment is of type string. This is a primitive type.

Reply:
@s77rt thank you for very detailed review of this PR 😊

`comment` here is actually the following type

```ts
{
  ...
  comment: {
    comment: string;
  }
  ...
}
```

so based on our guideline, we have to do

```ts
type Comment = {
  commment: string;
}

{
  ...
  comment: Comment
  ...
}
```

Reply:
Ah I see that makes sense! In that case shouldn't we add `originalTransactionID` and `source` as they are a part of the comment object (both are strings).